### PR TITLE
edit-installation-docs

### PR DESCRIPTION
**Pull request by pivilartisant**

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -70,6 +60,27 @@ Congratulations! You have successfully installed **MassaStation** on your comput
...
+6. If prompted, enter your administrator password to authorize the installation.
+7. The installation will commence, and you will see a progress bar indicating the status.
+8. Once the installation is complete, you will receive a notification confirming the successful installation.
+
+#### GUI update:
+
+1. Before proceeding to the update of Massastation, run the following command to uninstall the package:
+
+```bash
+sudo dpkg -r massastation
```
Comment (Thykof) [inline INSTALLATION.md:79]: should we say to uninstall Massa Station from the Software Install program?

Comment (pivilartisant) [reply INSTALLATION.md:79]: I'll check the flow and make sure it works correctly

Comment (pivilartisant) [reply INSTALLATION.md]: Flow is working i'll update rest of document additions

**Review by Thykof — APPROVED** [review]

**Comment by pivilartisant** [comment]
@longgamma-massa in this repo we use "MassaStation" should it be changed to "Massa Station"

**Review by Thykof — APPROVED** [review]

File: INSTALLATION.md [hunk 2]
```diff
@@ -89,31 +100,31 @@ The application and all modules installed will be deleted from your desktop.
...
 1. Open the Terminal application on your MacOS system.
-2. Execute the following command in the terminal to download and run the **MassaStation** uninstaller script:
+2. Execute the following command in the terminal to download and run the **Massa Station** uninstaller script:
 
    ```bash
    /usr/local/share/massastation/uninstall.sh
    ```
 
-   This command will remove MassaStation and its associated files from your system.
+   This command will remove Massa Station and its associated files from your system.
```
Comment (mazmassa) [inline INSTALLATION.md:112]:
```suggestion
   This command will remove **Massa Station** and its associated files from your system.
```

File: INSTALLATION.md [hunk 3]
```diff
@@ -89,31 +100,31 @@ The application and all modules installed will be deleted from your desktop.
...
 3. Follow any prompts or instructions provided by the uninstaller script. This may involve confirming the removal and providing your password for administrative privileges.
-4. Once the uninstallation process is complete, you will receive a confirmation message indicating that **MassaStation** has been successfully uninstalled.
+4. Once the uninstallation process is complete, you will receive a confirmation message indicating that **Massa Station** has been successfully uninstalled.
 
-> **Note:** DNSMasq and Homebrew might have been installed on your system as dependencies for MassaStation. We do not remove these packages automatically as they may be used by other applications on your system.
+> **Note:** DNSMasq and Homebrew might have been installed on your system as dependencies for Massa Station. We do not remove these packages automatically as they may be used by other applications on your system.
```
Comment (mazmassa) [inline INSTALLATION.md:116]:
```suggestion
> **Note:** DNSMasq and Homebrew might have been installed on your system as dependencies for **Massa Station**. We do not remove these packages automatically as they may be used by other applications on your system.
```

File: INSTALLATION.md [hunk 4]
```diff
@@ -89,31 +100,31 @@ The application and all modules installed will be deleted from your desktop.
...
-2. Execute the following command in the terminal to download and run the **MassaStation** uninstaller script:
+2. Execute the following command in the terminal to download and run the **Massa Station** uninstaller script:
 
    ```bash
    sudo dpkg -r massastation
    ```
 
-   This command will remove MassaStation and its associated files from your system.
-3. Once the uninstallation process is complete, you will receive a confirmation message indicating that **MassaStation** has been successfully uninstalled.
+   This command will remove Massa Station and its associated files from your system.
```
Comment (mazmassa) [inline INSTALLATION.md:129]:
```suggestion
   This command will remove **Massa Station** and its associated files from your system.
```

**Review by mazmassa — APPROVED** [review]

**Pull request merged**